src/foundry/common/constants.d.mts

```diff
@@ -2000,64 +2000,103 @@ export const TEXT_ENRICH_EMBED_MAX_DEPTH: number;
  */
 export declare const REGION_EVENTS: Readonly<{
   /**
-   * Triggered when the shapes or bottom/top elevation of the Region are changed.
+   * Triggered when the Region Behavior becomes active, i.e. is enabled or created without being disabled.
+   * The event is triggered only for this Region Behavior.
    */
-  REGION_BOUNDARY: "regionBoundary";
+  BEHAVIOR_ACTIVATED: "behaviorActivated";
 
   /**
-   * Triggered when the behavior is enabled/disabled or the Scene its Region is in is viewed/unviewed.
+   * Triggered when the Region Behavior becomes inactive, i.e. is disabled or deleted without being disabled.
+   * The event is triggered only for this Region Behavior.
    */
-  BEHAVIOR_STATUS: "behaviorStatus";
+  BEHAVIOR_DEACTIVATED: "behaviorDeactivated"
 
   /**
-   * Triggered when a Token enters a Region.
+   * Triggered when the Region Behavior becomes unviewed, i.e. inactive or the Scene of its Region is unviewed.
+   * The event is triggered only for this Region Behavior.
    */
-  TOKEN_ENTER: "tokenEnter";
+  BEHAVIOR_UNVIEWED: "behaviorUnviewed"
 
   /**
-   * Triggered when a Token exists a Region.
+   * Triggered when the Region Behavior becomes viewed, i.e. active and the Scene of its Region is viewed.
+   * The event is triggered only for this Region Behavior.
    */
-  TOKEN_EXIT: "tokenExit";
+  BEHAVIOR_VIEWED: "behaviorViewed"
 
   /**
-   * Triggered when a Token is about to move into, out of, through, or within a Region.
+   * Triggered when the shapes or bottom/top elevation of the Region are changed.
+   */
+  REGION_BOUNDARY: "regionBoundary";
```

Comment:
This is still the first option in Foundry source. Could you keep the properties in order with what Foundry source has please? It makes it easier to diff with Foundry source.

It seems you sorted it alphabetically or something? I've noted this only here but it does apply everywhere.

Reply:
That'd be great, thanks.